install.sh

```diff
@@ -65,12 +70,18 @@ Repeat the password: "
   local repeat_retry="
 Passwords do not match! Try again.
 "
-
+local validation_prompt="
+Password must be at least 8 characters long !.
```

Comment:
```suggestion
Password must be at least 8 characters long!
```

Reply:
Could you please the recursion-related bug in more detail so that I could resolve it as soon as possible?